cmd/cephcsi.go

```diff
@@ -68,6 +68,11 @@ func init() {
 	flag.StringVar(&conf.NodeID, "nodeid", "", "node id")
 	flag.StringVar(&conf.PluginPath, "pluginpath", defaultPluginPath, "plugin path")
 	flag.StringVar(&conf.StagingPath, "stagingpath", defaultStagingPath, "staging path")
+	flag.BoolVar(
```

Comment:
Just wondering, but have you considered to just set the metadata in case it is available?

Reply:
Can you explain a bit more, I didn't get what you mean by `set metadata in case it is available`?

Reply:
@nixpanic ?

Reply:
what Niels meant was to create the metadata if it's available in the request. let's not have a flag for this one as it will be useful to have the metadata on the ceph resources. is that correct @nixpanic? or set the metadata if it's available and the flag is set to true?